starter/src/test/java/com/starter/fullstack/dao/InventoryDAOTest.java

```diff
@@ -36,7 +37,17 @@ public void setup() {
   public void tearDown() {
     this.mongoTemplate.dropCollection(Inventory.class);
   }
-
+  // Test the create function. Initialize inventory and then add it to mongo collection with the create function. Print the ID before and after to check for null
+  @Test
+  public void createTest() {
+    Inventory inventory1 = new Inventory();
+    inventory1.setName(NAME);
+    inventory1.setId("877241");
+    inventory1.setProductType(PRODUCT_TYPE);
+    System.out.print("The initial ID is: " + inventory1.getId());
+    this.inventoryDAO.create(inventory1);
+    System.out.println("After Create Method ID: " + inventory1.getId());
```

Comment:
Part of testing is also verifying that after you run the method under test, `create` in this case, by using `assert`s